cmd/boost/deals.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+
+	bcli "github.com/filecoin-project/boost/cli"
+	"github.com/google/uuid"
+	"github.com/urfave/cli/v2"
+)
+
+var dealCmd = &cli.Command{
+	Name:  "deal",
+	Usage: "Manage Boost deals",
+	Subcommands: []*cli.Command{
+		makeOfflineDealWithData,
+	},
+}
+
+var makeOfflineDealWithData = &cli.Command{
+	Name:  "offline-deal-with-data",
+	Usage: "Make offline deal with data",
+	Flags: []cli.Flag{&cli.StringFlag{
+		Name:  "dealUuid",
+		Usage: "uuid of the offline deal",
+	},
+
+		&cli.StringFlag{
+			Name:  "filepath",
+			Usage: "path of the file containing the offline deal data",
+		},
+	},
+
+	Action: func(cctx *cli.Context) error {
+		if cctx.NArg() != 2 {
+			return fmt.Errorf("must specify deal uuid and deal data file path")
+		}
```

Comment:
Instead of doing this I think you can set `Required: true` on the flags

Reply:
Done.